src/controllers/eventController.ts

```diff
@@ -0,0 +1,18 @@
+// import EventModel from "../models/events";
+
+// class EventController {
```

Comment:
you can uncomment this and mock using the data above.

Reply:
i was still tackling some errors locally and i didnt want to push an error-filled file that doesnt work or pass the test

Reply:
also this doesn't have to be a class when all the methods will be static, you can just define the functions and export them as usual files.

e.g:
```suggestion
async getEvents(req:any, res:any, next:any) {
//     try {
//       const events = await EventModel.getEvents(1,"Community Challenge","Sirikwa Hotel",'2019-08-15 21:05:15.723336+07','2019-08-15 21:05:15.723336+07',"https://photos.app.goo.gl/323EcdkmFTzrfLd96","Marvin Kweyu", "chksavjhcb");
       const data = [...events];
       res.status(200).json({
        status: "success",
        data
     });
     } catch (error) {
      next(error);
     }
   }
 }

exports.eventsController = {
      getEvents
}
```

To import getEvents to use:

```
import { getEvents } from 'event-routes.ts'
```